codegen/lib/graphql_java_gen/templates/APISchema.java.erb

```diff
@@ -211,14 +212,14 @@ public class <%= schema_name %> {
                       children.add(this);
                     <% end %>
                     <% type.fields.each do |field| %>
```

Comment:
By looping through the schema fields rather than response keys I think we are also missing aliased fields.

Instead, maybe we should implement this in java by just looping over the response entries checking for values that are an instance of List or AbstractResponse, recursively, to get all the Node objects.  That would also reduce the amount of generated code.